modules/n1ql/pages/n1ql-language-reference/build-index.adoc

```diff
@@ -36,62 +39,161 @@ Index building might never complete if you get error 12014.
 To work around the issue, you can drop the indexes returning error 12014 and then recreate all of them either with a deferred build in one BUILD INDEX command (instead of multiple BUILD INDEX commands) or recreate them one at a time with a delay between the subsequent index creations to allow for index building to complete (that is index to get to the online state).
```

Comment:
I don't know if this workaround is a carry over from an older release. It should be sufficient to just wait for the in progress index builds to finish and then issue Build Index again. There is no need to drop and recreate the index.

Reply:
Done.

@orthogonal3 said:

> it looks like the issue was rectified as [MB-25155](https://issues.couchbase.com/browse/MB-25155)